examples/systemd/before-remove

```diff
@@ -4,9 +4,59 @@
 
 set -eu
 
-if [ $# -ge 1 ] && [ "$1" = "1" ]; then
-  echo "Skipping symlink removal as this is a package upgrade."
-else
+version_le() {
+  if command -v dpkg &>/dev/null; then
```

Comment:
```suggestion
  if command -v dpkg >/dev/null 2>&1; then
```
*nit*: &> is non-posix, bash 4+. You can ignore if all target systems have bash 4+.